tinygrad/renderer/llvmir.py

```diff
@@ -28,6 +28,7 @@
 
 def cast(bb, val, input_type, output_type):
```

Comment:
Can you write tests for this function?

Reply:
tests done, do you mean removing bf16 support completely?

Reply:
i removed bf16 - lmk if i need to add it back